src/wp-admin/about.php

```diff
@@ -39,60 +39,77 @@
 			</nav>
 		</div>
 
-		<hr />
+		<hr class="is-large" />
 
 		<div class="about__section">
 			<h2 class="aligncenter">
-				<?php _e( 'Three Essential Powerhouses' ); ?>
+				<?php _e( 'Full Site Editing is here' ); ?>
 			</h2>
+			<p class="aligncenter is-subheading">
+				<?php _e( 'It puts you in control of your whole site, right in the WordPress Admin.' ); ?>
+			</p>
 		</div>
 
+		<hr />
+
 		<div class="about__section has-2-columns is-wider-left">
-			<div class="column about__image is-vertically-aligned-center">
-				<img src="https://s.w.org/images/core/5.8/about-widgets-blocks.png" alt="" />
+			<div class="column about__image is-vertically-aligned-center is-edge-to-edge">
+				<img src="https://make.wordpress.org/core/files/2021/12/about-59-tt2.png" alt="" />
```

Comment:
Missing an alt value ?

Reply:
These images are considered decorative, since the text in the page describes the features.